jcl/source/common/JclSysInfo.pas

```diff
@@ -3368,8 +3369,13 @@ function GetWindowsVersion: TWindowsVersion;
                 else
                   Result := wvWinServer2012R2;
               end;
+          end;
         end;
-        end;
+        10:
```

Comment:
Windows 10 kernel version is 10.0 in Windows 10 Technical Preview Build 9926. It was 6.4 in previous builds.